namesrv/src/main/java/org/apache/rocketmq/namesrv/routeinfo/RouteInfoManager.java

```diff
@@ -156,6 +156,10 @@ public RegisterBrokerResult registerBroker(
 
                 BrokerData brokerData = this.brokerAddrTable.get(brokerName);
                 if (null == brokerData) {
+                    if (topicConfigWrapper.getTopicConfigTable().size() == 1) {
```

Comment:
I think it is better to judge Greater than or equal to 1, `topicConfigTable` maybe greater than 1 when batch send message

Reply:
I think this has nothing to do with sending messages. This is just when a topic is created, the broker will register the topic's routing information with namesrv, and it will only be 1. When the broker is fully registered, the size must be greater than 1, because the number of system topics has exceeded 1, right?

Reply:
Besides, If the judgment is greater than or equal to 1, it will be impossible to distinguish the full registration of the Broker from the registration which the topic is created

Reply:
Are you primarily trying to solve the problem of not being able to create a topic before first registering the broker

Reply:
> Are you primarily trying to solve the problem of not being able to create a topic before first registering the broker

I describe the current problem in detail in the issue. If someone creates a topic before the broker first registers with a new namesrv, the broker's first registration will fail. The registration can only be completed when the data version of TopicConfigManager is changed.

Reply:
Ok, I get it , Thanks for your reply